### PR TITLE
Port over adapters

### PR DESCRIPTION
**Pull request by dlrice**
# Note

- Yalc link the changes from this PR https://github.com/ebi-webcomponents/nightingale/pull/308
- `yarn jest` gives me these warnings
```
jest-haste-map: duplicate manual mock found: uniprotkb-entry-data
  The following files share their name; please delete one of them:
    * <rootDir>/dist/es/adapters/__mocks__/uniprotkb-entry-data.js
    * <rootDir>/dist/es/adapters/__tests__/__mocks__/uniprotkb-entry-data.js
```
but I couldn't figure out how to remove them. Leaving for now as they don't impact anything. Suggestions welcome.
- Created [jira to remove protvista-table](https://www.ebi.ac.uk/panda/jira/browse/TRM-32088) for future work.

### Description of changes

- Moved adapters into own directory
- Move tooltips into own directory
- Standardize all files to be kebab-case
- Use more types from nightingale
- Remove unused code
- Use lit to register web components
- Update exports to remove unused and add tooltips

### Testing/Styleguide
 - [x] Tests pass
 - [x] Linters pass

### CODE REVIEW

File: src/protvista-uniprot.ts [hunk 1]
```diff
@@ -124,7 +101,7 @@ type ProtvistaCategory = {
   'color-range'?: string;
 };
 
-export type DownloadConfig = {
+type DownloadConfig = {
```
Comment (Swaathik) [inline src/protvista-uniprot.ts:104]: We do not have download in protvista-uniprot anymore. we can delete it

**Review by Swaathik — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -36,26 +34,24 @@
     "@nightingale-elements/nightingale-structure": "5.2.0",
     "@nightingale-elements/nightingale-track": "5.2.0",
     "@nightingale-elements/nightingale-variation": "5.2.0",
+    "color-hash": "^2.0.2",
```
[the remark this answers is not in the document]
Comment (dlrice) [reply package.json]: Pinned all dep versions but not devDeps as I encountered webpack issues so left as is there.

File: src/protvista-uniprot.ts [hunk 3]
```diff
@@ -14,68 +16,43 @@ import NightingaleLinegraphTrack from '@nightingale-elements/nightingale-linegra
...
+import interproAdapter from './adapters/interpro-adapter';
+import variationGraphAdapter from './adapters/variation-graph-adapter';
+import proteomicsPTMApdapter from './adapters/ptm-exchange-adapter';
+import alphaFoldConfidenceAdapter from './adapters/alphafold-confidence-adapter';
+import alphaMissensePathogenicityAdapter from './adapters/alphamissense-pathogenicity-adapter';
+import alphaMissenseHeatmapAdapter from './adapters/alphamissense-heatmap-adapter';
+
+import ProtvistaUniprotStructure from './protvista-uniprot-structure';
+
+import { load } from 'data-loader';
```
[the remark this answers is not in the document]
Comment (dlrice) [reply src/protvista-uniprot.ts]: Removed

File: src/protvista-uniprot.ts (hunk 1 above)
Comment (dlrice) [reply src/protvista-uniprot.ts]: Removed

**Review by Swaathik — APPROVED** [review]

**Pull request merged**